gemini-resin-archetype/src/main/resources/archetype-resources/src/main/webapp/WEB-INF/web.xml

```diff
@@ -45,6 +45,7 @@
   <servlet>
     <servlet-name>Server</servlet-name>
     <servlet-class>com.techempower.Server</servlet-class>
+    <load-on-startup/>
```

Comment:
I have never included this but it also seemed like this was true as a default... am I crazy?

Reply:
Happy to discard this. My preference is for the app to start on its own so I don't have to wait when I make my first request.

Documentation: https://stackoverflow.com/a/7817279